source/clear-linux/get-started/virtual-machine-install/vmware-esxi.rst

```diff
@@ -1,80 +1,520 @@
 .. _vmware-esxi:
 
-Use VMware\* ESXi guest
-#######################
+Run Clear Linux as a VMware\* ESXi guest OS
+###########################################
 
+VMware ESXi is a type 1 bare-metal hypervisor which runs directly on top of 
+server hardware.  Creation, configuration, management, and execution of virtual
+machines are done in the cloud.  For the desktop version of VMware, see 
+:ref:`vmware-player`.
 
-This section is based on *VMware vSphere 6* and explains how to use |CLOSIA|
-as ESXi Guest.
+This section explains how to run |CLOSIA| in a virtualized environment using 
+VMware ESXi 6.5 Update 1.  
 
-Ensure `Intel® Virtualization Technology`_ (Intel® VT) and
-`Intel® Virtualization Technology for Directed I/O`_ (Intel® VT-d) are both
-enabled in your BIOS/UEFI firmware configuration.
+There are 2 ways to create a |CL| :abbr:`VM (Virtual Machine)` to run in VMware:
 
-Run Clear Linux OS for Intel Architecture
-=========================================
+* `Method 1`: Install |CL| into a new VM.  This provides flexibility 
+  in configuring the VM size, partitions, initial bundles installation, etc.
+* `Method 2`: Use a ready-made VMware |CL| image.
 
-#. Download the `latest`_ |CL| **live** version
-   :file:`clear-{release-number}-live.img.xz`.
+Both methods are discussed below.
```

Comment:
If you are going to structure the tutorial by which image is used, which I would not recommend, then the next section must be titled "Method 1" and have all the instructions from beginning to end using that image. I recommend you get rid of this line.